templates/multiplatform-library-template/library/build.gradle.kts

```diff
@@ -0,0 +1,84 @@
+import org.jetbrains.compose.compose
+
+plugins {
+    id("com.android.library")
+    kotlin("multiplatform")
+    id("org.jetbrains.compose")
+    id("maven-publish")
+}
+
+group = "com.mylibrary"
+
+kotlin {
+    android {
+        publishLibraryVariants("release")
+    }
+    jvm("desktop")
+
+    sourceSets {
+        named("commonMain") {
+            dependencies {
+                api(compose.runtime)
+                api(compose.foundation)
+                api(compose.material)
+                // Needed only for preview
+                implementation(compose.preview)
```

Comment:
I think these dependencies should `compileOnly` to make the library also compatible with Android only projects.